src/task 33/main.js

```diff
@@ -0,0 +1,24 @@
+var tArea = document.createElement('textarea');
+tArea.value = 'return "Hi"';
+tArea.id = "code";
+document.body.append(tArea);
+var butt = document.createElement('button');
+butt.innerHTML = 'Go';
+butt.id = "button"
+document.body.append(butt);
+var pre = document.createElement('pre');
+pre.id = "output";
+document.body.append(pre);
+
+butt.addEventListener("click", function(){
+    var JS = tArea.value;
```

Comment:
Better create a separate reusable function with passed hard-coded configs and then use it as event handler. So you could easily re-use it later.